character-quiz/index.ts

```diff
@@ -81,9 +81,9 @@ const loadCharacters = async (author: string) => {
 			const characterNames = characterName.split('、').filter((name) => name !== '');
 			const characterRubys = characterRuby.split('、').filter((name) => name !== '');
 
-			if (characterNames.length === 0 || characterRubys.length === 0) {  
-				return [] as CharacterData[];  
-			}  
+			if (characterNames.length === 0 || characterRubys.length === 0) {
```

Comment:
Revert the unnecessary changes in character-quiz and functions. Make sure you revert these files every time before you commit changes.